index.js

```diff
@@ -10,39 +10,37 @@ function scrollToTop() {
   }
 }
 $(document).ready(function() {
-  const $slider = $(".slider-inner");
+  const $sliderInner = $(".slider-inner");
```

Comment:
`sliderInner` is too general name. It's really hard to distinguish one inner slider from another.

> There are only two hard things in Computer Science: cache invalidation and naming things.
> 
> -- Phil Karlton

I suggest you to think about how would you separate this slider from another slider so it's clear for you and other people. Uniqueness is a key. You can't call it `inner` since it's not clear inside what it is.

Bad name: `sliderOne` (it might be removed so there will be only `sliderTwo` is not straightforward), `sliderRight` (slider might be moved to another place so it won't be right anymore, or another slider might be added at the right so it would be hard to distinguish them), `sliderTop` (the same thing as `sliderRight`), `slider`(no separation), `mainSlider`(it's no obvious why one slider is main on this site)
A bit better names: `linensSlider`, `promoSlider`, `linensPromoSlider`

I would go with `promoSlider` and `bestSellersSlider`. It make a lot of sense to me at this point of time

Reply:
Thanks for the useful information! You are the best programmer for me!